xc7/utils/prjxray_tile_import.py

```diff
@@ -1396,10 +1401,10 @@ def main():
     )
 
     parser.add_argument(
-        '--both_site_coords',
-        action="store_true",
-        default=False,
-        help="""Use both site coordinates for the fasm prefix."""
+        '--site_coords',
+        type=str,
+        default='X',
+        help="""Specify which site coords to use ('X', 'Y' or 'XY')"""
```

Comment:
You can use [choices](https://docs.python.org/3/library/argparse.html#choices) to enforce the options at argparse time.